### ISSUE TITLE
Issue with Spanish-language characters in VCE pubs

### ISSUE THREAD

**Issue by mello99** [VTUL/vtechworks#256]
Hi, @alawvt , I was talking to Jana earlier about language encoding and she showed me a VCE record in Spanish - https://vtechworks.lib.vt.edu/handle/10919/50757. The issue with this record is that the Spanish-language characters are awry, and we were trying to ascertain why this happened. We think that this item may have been batch loaded, but we're not positive. Any ideas? Thanks so much!

**Comment by alawvt** [comment]
@mello99, I loaded that in batch, Food_Safety: 20140814_10919_5548, which I uploaded on 2014-11-05. Attached is the metadata file that was uploaded for that item. (I changed the file name from dublin_core.xml to dublin_core.txt so GitHub would upload it.) It looks ok in BBEdit which says it has UTF-8 encoding. 
[dublin_core.txt](https://github.com/VTUL/vtechworks/files/242991/dublin_core.txt)  
Since I had the metadata file open, I fixed the fields for this item.  
I just ran a test batch with just this item and loaded it to VTechWorks; then deleted the item. The metadata encoding was fine for it in VTechWorks. So, I don't know why this one had problems, but it doesn't appear to be something that is systematically occurring now. One other thing to note is that this was loaded in either DSpace 3 or 4.
